modules/backend/layouts/auth.php

```diff
@@ -28,14 +28,14 @@
         ];
         ?>
 
-        <?php foreach ($styles as $style): ?>
-            <link href="<?= $style . '?v=' . $coreBuild; ?>" rel="stylesheet" importance="high">
-            <link href="<?= $style . '?v=' . $coreBuild; ?>" rel="preload" as="style" importance="high">
+        <?php foreach ($styles as $style) : ?>
+            <link rel="preload" href="<?= $style . '?v=' . $coreBuild; ?>" as="style">
+            <link rel="stylesheet" href="<?= $style . '?v=' . $coreBuild; ?>"/>
         <?php endforeach; ?>
 
-        <?php foreach ($scripts as $script): ?>
-            <script data-cfasync="false" src="<?= $script . '?v=' . $coreBuild; ?>" importance="high"></script>
```

Comment:
`data-cfasync` is still required AFAIK is it not? It's to prevent CloudFlare's Rocket Loader from breaking the backend.

Reply:
Doesn't seem to be doing anything, nothing is broken without it

Reply:
It will break sites for people who use CloudFlare Rocket Loader, but ideally, I would like for us to not have platform-specific fixes in core code and delegate it to a plugin, if it all possible.

Reply:
Normally we do handle it via a plugin (https://github.com/wintercms/wn-cloudflare-plugin/blob/main/Plugin.php#L43-L54) but these are different because they're manually included instead of going through the AssetMaker trait.

Reply:
Is it worth us changing these to go through AssetMaker then?

Reply:
@bennothommo if you feel like doing that, sure 😉